prompt.go

```diff
@@ -49,6 +62,115 @@ func (i *InitActionCmd) setParams(dirs []string) error {
 			}
 		}
 	}
+	if i.ConfigPath == "" && len(configs) > 0 {
+		switch {
+		case len(configs) == 1:
+			content, err := repo.ReadContent(ctx, configs[0])
+			if err != nil {
+				return err
+			}
+			file, diags := hclparse.NewParser().ParseHCL([]byte(content), "atlas.hcl")
+			if len(diags) > 0 {
+				return fmt.Errorf("failed to parse %s: %s", i.ConfigPath, diags.Error())
+			}
+			var envs = make(map[string]bool)
+			for _, b := range file.Body.(*hclsyntax.Body).Blocks {
+				if b.Type == "env" {
+					_, hasDev := b.Body.Attributes["dev"]
+					envs[b.Labels[0]] = hasDev
+				}
+			}
+			if len(envs) > 0 {
+				opts := []string{"no"}
+				envNames := make([]string, 0, len(envs))
+				for k := range envs {
+					envNames = append(envNames, k)
+				}
+				opts = append(opts, envNames...)
+				prompt := promptui.Select{
+					Label:    fmt.Sprintf("Use %q file as config?", configs[0]),
+					HideHelp: true,
+					Items:    opts,
+					Templates: &promptui.SelectTemplates{
+						Active:   "{{ if eq . \"no\" }}▸ no{{ else }}▸ use with env {{ . | bold }}{{ end }}",
+						Inactive: "{{ if eq . \"no\" }}  no{{ else }}  use with env {{ . | bold }}{{ end }}",
```

Comment:
Why not use regular options? 
```suggestion
						Active:   "{{ if eq . \"no\" }}▸ No{{ else }}▸ Use with env {{ . | bold }}{{ end }}",
						Inactive: "{{ if eq . \"no\" }}  No{{ else }}  Use with env {{ . | bold }}{{ end }}",
```

Reply:
Maybe also keep `Use ...` first and move `No` to the second option.